modules/commons/src/main/resources/schemas/neurosciencegraph/commons/variables/v0.2.0.json

```diff
@@ -0,0 +1,136 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/variables/v0.2.0/shapes/"
```

Comment:
We might want to rename it to commons/variable/v0.2.0 to have it consistent with other schemas (i.e. schema name singular)?

Reply:
See comment made for `selectors`.